docker-compose.yml

```diff
@@ -1,7 +1,8 @@
-# docker-compose.yml
-version: '3'
+version: '3.8'
+
 networks:
   todoapp-net:
+
 services:
   pythonapp:
     image: todoapp:2.0.0
```

Comment:
The `image` tag specifies `todoapp:2.0.0`. Ensure that this tag is correctly built and pushed to a Docker registry if you intend to use it. If you are building locally, this line might be unnecessary since the `build` context is provided.